### ISSUE TITLE
Square video format

### ISSUE THREAD

**Issue by denisokvrn** [natario1/CameraView#17]
Can I change library for get square video as in instagram?

**Comment by UGhari** [comment]
Yes you can, read the documentation. The issues section isn't an area where you ask for tutorials.

**Comment by denisokvrn** [comment]
sorry. in documentation i find only square preview. but output video file has rectangle size (not square as in preview)

**Comment by natario1** [comment]
Yes @denisokvrn you are right. This will be supported in the next few days

**Comment by Mack4545** [comment]
@natario1 Thanks so much I would really appreciate this enhancement!

**Comment by Mack4545** [comment]
@natario1 Hello. I didn't see this enhancement in the commit notes 3 days ago... is this something you think you're going to still do? Thanks.

**Comment by natario1** [comment]
@Mack4545 sure, just be patient please, I will be very busy next week.

**Comment by jays95jp** [comment]
@natario1  really appreciate your effort and for this enhancement. one more requirement for me want to record video like Instagram , Thanks

**Comment by cyfer** [comment]
Hi @natario1 , thanks for the great library! Is there any update on being able to record Square videos? I have seen on another thread a user proposing the use of ffmpeg to achieve this, but I would prefer a “native” solution if possible.

**Comment by natario1** [comment]
Thanks @cyfer ! I don't know, but subscribe to this issue to get updates

**Comment by skined90** [comment]
Is there any news on this one?
